content/trading-on-alpaca/account-plans.md

```diff
@@ -43,6 +43,9 @@ Today’s beta program for the business trading account accepts select entities
 
 You can [sign up](https://app.alpaca.markets/signup) with your email address and follow steps on the dashboard. When our system considers that your business entity is eligible for this beta program, you should be able to go on to upload necessary information through the process to submit the application, otherwise you will be placed on the waitlist. Please note that we are updating the beta eligibility criteria and business trading account opening flow with ongoing basis.
 
+## Alpaca Cryptocurrency Account (Live-Trading)
+
+We currently offer cryptocurrency trading to users in California, Massachusetts, Missouri, and Montana. If you are in those states, crypto trading can be enabled while creating your Alpaca Brokerage Account by accepting the Crypto Agreement. If you already have an Alpaca Brokerage Account, you can enable crypto trading by updating the crypto-enabled setting in Account Configuration through the Alpaca web dashboard or the [API](https://alpaca.markets/docs/api-documentation/api-v2/account-configuration/).
```

Comment:
@excavator20  --- I left off Hungary, but if I should mention it I can add it.

Reply:
Also, if I recall correctly there's a slider in the Account Configuration to enable crypto--- and I'm assuming it triggers the crypto agreement acceptance flow. I didn't see any value in the account configurations endpoint though yet on develop branch, but I'm also assuming it will be PATCHable there. I'll change crypto-enabled to whatever the real field name is.